airbyte-integrations/connectors/source-facebook-pages/source_facebook_pages/components.py

```diff
@@ -83,12 +83,12 @@ class NestedDpathExtractor(DpathExtractor):
 
     def extract_records(self, response: requests.Response) -> List[Record]:
         response_body = self.decoder.decode(response)
-        if len(self.field_pointer) == 0:
```

Comment:
`NestedDpathExtractor` seems to be something that would be interesting in the airbyte-cdk per se. If we agree, I could create a Continuous Improvement ticket to migrate it (or make DpathExtractor support the same functionality)

Note: To be clear, I haven't checked much the difference and I'm not clear on what the gap between `DpathExtractor` and `NestedDpathExtractor` is

Reply:
I think we can deprecate the nested one: https://github.com/airbytehq/airbyte/pull/21690

Reply:
Confirmed with artem1205 that GL will remove this eventually